MTM-51772 revoke tokens to log out users (#934)
SoftwareAG/c8y-docs: content/users-guide/administration-bundle/managing-users.md

You need a role with user management permission to perform this option.
{{< /c8y-admon-info >}}

### To log out all users

In the event of a security incident involving the session tokens of your tenant's users, you can invalidate any tokens currently in use.

> To log out all users click **Log out all users**  at the right of the top menu bar. This will log out all users logged with "OAI-Secure" or "Single sign-on redirect". JWT tokens retrieved by all devices in the current tenant will also be invalidated.

Reviewer: ```suggestion
To log out all users click **Log out all users**  at the right of the top menu bar. This logs out all users currently logged in via OAI-Secure or single sign-on redirect. JWT tokens retrieved by all devices in the current tenant are also invalidated.
```

> Note that, if the "Basic Authentication" is used, the users logged with base64 token will logged out.

Reviewer 2: I guess it is not true.

> Note that, if the "Basic Authentication" is used, the users logged with base64 token will logged out.

Reviewer: ```suggestion
Note that, if basic authentication is used, users logged in via base64 token are logged out.
```

> Note that, if the "Basic Authentication" is used, the users logged with base64 token will logged out.

Reviewer: ```suggestion
Note that, if the "Basic Authentication" is used, the users logged with base64 token are logged out.
```
